src/TLuaInterpreter.cpp

```diff
@@ -7161,6 +7171,39 @@ int TLuaInterpreter::setConfig(lua_State * L)
         host.mAdvertiseScreenReader = getVerifiedBool(L, __func__, 2, "value");
         return success();
     }
+
+    static const QStringList behaviours{"donotshare", "share", "block"};
```

Comment:
Remember, no unwrapped raw C string literals in `QString`s:
```suggestion
    static const QStringList behaviours{qsl("donotshare"), qsl("share"), qsl("block")};
```

Reply:
Done